hbase-server/src/test/java/org/apache/hadoop/hbase/master/cleaner/TestSnapshotFromMaster.java

```diff
@@ -530,8 +530,11 @@ public boolean evaluate() throws Exception {
         return UTIL.getHBaseAdmin().listSnapshots(Pattern.compile(snapshotName)).size() == 1;
       }
     });
-    assertTrue(master.getSnapshotManager().isTakingAnySnapshot());
-    Thread.sleep(11 * 1000L);
-    assertFalse(master.getSnapshotManager().isTakingAnySnapshot());
+    UTIL.waitFor(30000, new Predicate<Exception>() {
+      @Override
+      public boolean evaluate() throws Exception {
+        return master.getSnapshotManager().isTakingAnySnapshot();
```

Comment:
Should add a "!" here? It is assertFalse before.

Reply:
Oh right! My bad. Overlooked the condition.

Reply:
Corrected.